devtools-ci-index.yaml

```diff
@@ -473,6 +473,20 @@
                 credential-id: 2fc94f6b-dfef-49a4-a217-fe47aaeea056
                 variable: RECOMMENDER_API_TOKEN
 
+- wrapper:
+    name: copr_mercator_api_token
+    wrappers:
+        - credentials-binding:
+            - text:
+                credential-id: ???
+                variable: COPR_LOGIN_MERCATOR
+            - text:
+                credential-id: ???
+                variable: COPR_USERNAME_MERCATOR
```

Comment:
Is `login` and `username` a secret?

Reply:
username no, but login is some hash / random string